src/components/CardList/Card/index.js

```diff
@@ -127,4 +145,11 @@ Card.propTypes = {
   viewMode: PropTypes.bool
 }
 
-export default withLoadingDelay(Card);
+const mapDispatchToProps = dispatch => {
+  return {
+    onCheck: (id, checked) => dispatch(actionCreators.toggleCard(id, checked)),
+    onSave: (id, tempData) => dispatch(actionCreators.saveChanges(id, tempData))
+  }
+}
```

Comment:
You can try out react-redux hooks since you're mostly working with functional components. I'm pretty sure, you'' like it.
https://react-redux.js.org/api/hooks